app/models/vehicle.rb

```diff
@@ -0,0 +1,6 @@
+class Vehicle < ApplicationRecord
+  has_many :bookings, dependent: :destroy
+  has_many :drivers, dependent: :destroy
+  has_many :rides, dependent: :destroy
+  belongs_to :category
+end
```

Comment:
Please kindly also add the relationship with the parking slot as shown in the diagram